ingest/src/svm.rs

```diff
@@ -151,6 +177,8 @@ impl<'py> pyo3::FromPyObject<'py> for D8 {
 #[cfg_attr(feature = "pyo3", derive(pyo3::FromPyObject))]
 pub struct InstructionRequest {
     pub program_id: Vec<Address>,
+    pub data: Vec<Data>,
+    pub discriminator: Vec<Data>,
```

Comment:
I'm creating both a data and a discriminator; they work exactly the same. Let me know if you think this is confusing.

Reply:
discriminator is good imo, can remove data